functional-tests/devnet-retrieval

```diff
@@ -125,19 +123,13 @@ echo "miner node starts mining..."
 ./go-filecoin mining start \
   --repodir="$MN_REPO_DIR"
 
-echo ""
-echo "Waiting until messages are in blockchain..."
-fork_message_wait 104 "${MINER_UPDATE_PID_MSG_CID}" "${MN_REPO_DIR}"
-fork_message_wait 105 "${MINER_UPDATE_PID_MSG_CID}" "${CL_REPO_DIR}"
-fork_message_wait 106 "${MINER_ADD_ASK_MSG_CID}" "${MN_REPO_DIR}"
-fork_message_wait 107 "${MINER_ADD_ASK_MSG_CID}" "${CL_REPO_DIR}"
+MINER_ADD_ASK_MSG_CID=$(set_price 10 10000 "${MN_REPO_DIR}")
 
 echo ""
-echo "block until miner peer id-update message appears in chains..."
-join 104
-join 105
-join 106
-join 107
+echo "waiting until messages are in blockchain..."
+./go-filecoin message wait "${MINER_UPDATE_PID_MSG_CID}" --repodir="${MN_REPO_DIR}"
+./go-filecoin message wait "${MINER_UPDATE_PID_MSG_CID}" --repodir="${CL_REPO_DIR}"
+./go-filecoin message wait "${MINER_ADD_ASK_MSG_CID}" --repodir="${CL_REPO_DIR}"
```

Comment:
maybe note the reason we dont have to wait on the miner node for the add ask, presumably bc there is a wait in set price above

Reply:
Done